tests/system_tests_distribution.py

```diff
@@ -3910,11 +3908,9 @@ def send_from_client(self, sender, n_messages, sender_index):
             n_sent             += 1
             self.n_sent        += 1
             self.n_transitions += 1
-            self.debug_print("send_from_client -- sender: %d n_sent: %d" % (sender_index, n_sent))
+        self.debug_print("send_from_client -- sender: %d n_sent: %d" % (sender_index, n_sent))
```

Comment:
Did you also want to do the same thing (move the debug print out of the loop) for the send_from_waypoint() method below?  Just asking...

Reply:
Actually, no. The loop in send_from_waypoint prints the message_content_number, so I want to keep the print inside the loop